Gems/Atom/Feature/Common/Code/Source/DiffuseGlobalIllumination/DiffuseProbeGridBlendDistancePass.cpp

```diff
@@ -30,10 +31,11 @@ namespace AZ
         DiffuseProbeGridBlendDistancePass::DiffuseProbeGridBlendDistancePass(const RPI::PassDescriptor& descriptor)
             : RPI::RenderPass(descriptor)
         {
-            if (!AZ_TRAIT_DIFFUSE_GI_PASSES_SUPPORTED)
+            if (!(RHI::RHISystemInterface::Get()->GetDevice()->GetFeatures().m_rayTracing && AZ_TRAIT_DIFFUSE_GI_PASSES_SUPPORTED))
```

Comment:
Please remove the m_rayTracing check from this class (and the other non-RT DiffuseProbeGrid passes).